### PR TITLE
release: update changes & release notes for v0.6.0 and v0.6.1

### PR DESCRIPTION
**Pull request by iGormilhit**
Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?

- To document RERO ILS `v0.6.0` and `v0.6.1`.

## How to test?

- See the files diff.
- Check also the rendering of [CHANGES.rst](https://github.com/iGormilhit/rero-ils/blob/060-release-changes/CHANGES.rst) and [RELEASE-NOTES.rst](https://github.com/iGormilhit/rero-ils/blob/060-release-changes/RELEASE-NOTES.rst)

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,243 @@
...
+
+-  Updates JSON schemas to the new ``ngx-formly`` library (form options
+   are now directly in the schema).
+-  Sets the document abstract field to textarea type.
+
+Acquisition
+-----------
+
+-  Adds new resources for vendors file, orders and order lines.
+-  Adds new resources for acquisition budget and acquisition account.
```
Comment (BadrAly) [inline RELEASE-NOTES.rst]: Adds new resources for the organisation acquisition budget and the library acquisition accounts

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,243 @@
...
+-  Sets the acquisition account currency from organisation default
```
Comment (BadrAly) [inline RELEASE-NOTES.rst]: The library accounts inherits the currency from its organisation

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,243 @@
...
+   ``invenio utils runindex``).
+-  Improves persons indexing during document indexing and document
+   creation.
+-  Fixes missing mappings in JSON schemas.
+-  Adds default sorting for each resource.
+
+Circulation
+-----------
+
+-  Adds a method to get the timezone of a library to improves
```
Comment (BadrAly) [inline RELEASE-NOTES.rst]: The circulation transactions, including checkouts, now take into account the timezone of the library dates

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,243 @@
...
+   computation of a checkout due date.
+-  Links loans and fees.
```
Comment (BadrAly) [inline RELEASE-NOTES.rst]: The links between the resources fee and loan are now made via the resource notification

File: RELEASE-NOTES.rst (hunk 2 above)
Comment (iGormilhit) [reply RELEASE-NOTES.rst]: Yes, but I'd like to have only sentences beginning with a verb, 3rd singular person.

**Pull request merged**